src/Sarif/Sarif.csproj

```diff
@@ -29,17 +29,26 @@
     <None Remove="NotYetAutoGenerated\**" />
   </ItemGroup>
 
+  <Choose>
+    <When Condition="$(TargetFramework) == 'netstandard2.0'">
+      <ItemGroup>
+        <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+      </ItemGroup>
+    </When>
+    <Otherwise>
+      <ItemGroup>
+        <PackageReference Include="Newtonsoft.Json" Version="6.0.8" />
+        <Reference Include="System.Web" />
+        <Reference Include="System.Net.Http" />
+      </ItemGroup>
+    </Otherwise>
+  </Choose>
```

Comment:
Is Newtonsoft.Json guaranteed to be compatible with all of our supported versions besides netstandard 2.0?

Also, glad to see an example of this. @EasyRhinoMSFT, is this along the lines of what you were experimenting with the other week?

Reply:
Yes, compatible.